.env.development.example

```diff
@@ -1,4 +1,8 @@
 ADDRESS_SERVER_URL=https://beta.3box.io/address-server
 ORBITDB_PATH=./dev-data/orbitdb
 IPFS_PATH=./dev-data/ipfs
-#REDIS_PATH=127.0.0.1
+REDIS_PATH=127.0.0.1
+AWS_BUCKET_NAME=experiment-3box-ipfs
+AWS_ACCESS_KEY_ID=AKIA0000000000000000
+AWS_SECRET_ACCESS_KEY=fG00000000000000000000000000000000000000
```

Comment:
What are these keys?

Reply:
S3 keys, just text placeholder example env vars, for a second I thought I committed my keys and GitHub was now zeroing out env keys hahah